resources/views/admin/messages/success.blade.php

```diff
@@ -0,0 +1,7 @@
+@if (Session::has('flash_message'))
```

Comment:
the path to this file should be `resources/views/layout/admin/partials/flash.blade.php`